### DISCUSSION TITLE
Offer of help in the form of Video Training

### DISCUSSION THREAD

**Discussion by paurray** [templaza/astroid-framework#361] [Show and tell]
Hello Astroid Community

In a response to this here:
**"We need you to grow Astroid together!"** https://github.com/templaza/astroid-framework/discussions/299
I have created 8 videos which run at a total of 83 minutes.

In these videos I run through building a website with Astroid on a step by step basis:

- In the interest of managing expectations….
- Straight off the bat I am neither a Joomla or Astroid expert.
- I have a long and chequered Joomla history.
- Have been using Astroid for **exactly a year now!**
- The videos are probably far from perfect
- But I hope they can help novices.
- And generally make Astroid more accessible.

Back Story:

- Coming from a video editor background.
- Decent amount of training experience here.
- Help from @Chacapamac who tried to keep me on the straight and narrow.
- Thank you for your patience.
- Who I hope has no objection to me mentioning him in this context?

I am currently uploading the Videos to a You Tube Channel which I have created:

- This is taking forever as the videos have a large size.
- The Channel is currently called "Paul Dust" but this is not carved in stone.
- I am an absolute You Tube Nob so this is all somewhat of an experiment for me.
- If I had to guess I can post more info on Monday next week.

The important thing for me:
- Is that I want to have some kind of control/copyright over the videos.
- Apart from that they are free although I will probably set up a Buy me a coffee link.

If any one watches them and thinks they are any good….

- Then maybe we could brain storm here how to help spread the word?
- I will post again when the videos are published.
- Or maybe I can set them up with a password first (which I will post here).
- If they are truly awful I can let them die on the vine or even remove them.

**I would greatly appreciate feed back and thoughts about moving forward.**

All the best

Paul

**Comment by sonvnn** [comment]
Hi @paurray

Thank you for your contribute. @Chacapamac May you help us give some feedback for his ideas? We can discuss more here.

Best Regards,
Sonny

**Comment by Chacapamac** [comment]
**I guys, sorry I’m out of the game for a while, it should come back to normal soon.**

Yes Paul have made a great deal of work on a great series of video to initiate people in making websites with Astroid and Joomla.
I just help a bit in the background. The result is a perfect initiation on how to start making websites (the right way) with both Astroid & Joomla. It also give an easy to follow overview of the power of Joomla and the Astroid Framework.

**I think that will should put this series under the existing Astroid YouTube Channel** ➔  https://www.youtube.com/channel/UCUHl1uU0Ofkyo-1ke-K4_xg under the already created playlist “**Astroid Joomla Framework Tutorials** or in a playlist by their own to be sure they are view together  — I suggest a name like “ How to make a website with Astroid & Joomla” or ”Your first website with Astroid & Joomla”

Ideally Paul or Me, or both should be made Managers of the Astroid Youtube Channel. Like that, Paul (or me) can insert ourself our own videos and organized the channel the right way.

@paurray , @sonvnn  — **What do You think?** 

**• Video how to make Managers/Administrators to a Youtube Channel**
https://www.youtube.com/watch?v=tn5E_cPYxLE

_**Funny,** on this video I can swear that this is the voice of Chetan Madaan (Owner of Joomdev)_

**Comment by paurray** [comment]
Good Morning,, Good Evening, Good Night
@sonvnn @Chacapamac and anyone who is interested.

Thanks for all the fabulous feedback so far. Greatly appreciated.

**Backing up a couple of steps first…**

Maybe someone would like to take the time to watch the videos they are here:
(The videos are not listed as in you can currently only access them using the links below. And can not be searched for or found on You Tube.)

01-Astroid-PREAMBLE-WAY-TOO-LONG
https://youtu.be/JB8Fsn6nd-g
02 Astroid INSTALL FIRST BABY STEPS
https://youtu.be/dWu_j6xLO2Y
03 Astroid JOOMLA CATEGORIES ARTICLES MENUS
https://youtu.be/fLVWoQeRfW4
04 Astroid ARTICLE CONTENT FORMATTING
https://youtu.be/Jq5rG5mcWKc
05 Astroid TYING UP FORMATTING
https://youtu.be/o93_ZQd-xVs
06 Astroid MODULE TYPES LAYOUT BUILDER
https://youtu.be/NoIIK5hHHxg
07-Astroid-FORMATTING-BROAD-BRUSH-STROKES
https://youtu.be/bWEQ4FfeUIU
08-Astroid-TYING-UP-LOSE-ENDS
https://youtu.be/dq-WH0XvQ14

My offer is pretty much a take to or leave it offer. 

Do the videos provide: 
**A net gain? Yes/No OR a net loss? Yes/No**

*The small print: I talked with a Lawyer friend and she said that having an account with Multiple Admins is a bad idea. The reason being that I/we have no control over what other admins do with it. It does not even have to be another admin doing something bad. It could be as banal as an admins access getting hacked. Something malicious happening and theoretically being as a fellow admin responsible.  But I am getting ahead of myself. Maybe just a playlist in the existing Astroid channel with an Admin other than me :-)

**Feedback by the end of the week. 04/08 would be a huge help** as it is going to get mad here as in going to be slammed with work.

Assuming Yes.

General Random thoughts moving forward:

General Random thoughts:

- Need to add descriptions to videos. (Updated / **DONE**)
- Need to add links in descriptions (sometimes) (Updated / **DONE**)
- Need to add chapter markers. (Updated / **DONE**)
- There is an option to add an end screen/might be something to think about.
- Can add cards linking to other videos during the videos.
- Very flexible about Playlist name and where it is created. (Updated / **DONE**)
- Would like to keep the name on the intro/outro graphics though. (Left) 
- Interested in things I missed eg stupid spelling mistakes.
- I do mention specific people in videos particularly the first one is this O?
- On YT there is a publish schedule option eg publish 1 a week for 8 weeks.

Thanks and I am pretty positive that we can find a way.
Paul

Edited to show updated tasks

**Comment by paurray** [comment]
(Updated)

Changed the name of the channel to
“Astroid & Joomla Seamlessly”

Did some very basic layout stuff:

https://www.youtube.com/@PaulDust

Created a playlist.

**Title:**
Your first “Bare Bones” website with Astroid & Joomla

**Description:**
Step by step instructions. How to build a website from scratch using Astroid and Joomla. Suitable for Novices.

https://www.youtube.com/playlist?list=PLOd7DxRu2yaJvVJKPEGZAOGoWpWOyNL1b

It is still unlisted. Will probably just publish a vide a week from next week onwards…

<img width="1005" alt="AstroidChannel" src="https://github.com/templaza/astroid-framework/assets/9163424/343872e2-ef87-4a61-a3c4-69f346173d02">

**Reply by Chacapamac** [reply]
> Changed the name of the channel to “Astroid & Joomla Seamlessly

Cool, Ideally these videos should also be available directly in the main Astroid Youtube Channel.

You got your first subscriber. 😀

**Comment by paurray** [comment]
I would be happy to have the videos as a Playlist inserted on the Astroid Chanel.
Would like to have the final word (as none admin) on: Thumbnail, Playlist Description & Settings.

From my job as an editor I am constantly editing videos that get posted online.
The amount of work and sometimes money that goes into them is insane.
And then they end up getting something like 18 hits.
I see this happen time and time again.
Frankly it is frustrating.
Which always makes me think why? Why bother?
I know very little about promoting videos as such

But maybe a more important question than where these videos are played is….

...how can they gain some traction? or returning to the original brief **"grow Astroid together"**

**Assuming that a fixed launch date X s decided:**

- Maybe Tweet about it.
- Post something on Facebook in a Joomla Group?
- Discord/Slack/Mastodon or what ever these things are called these days?
- Or does someone have a newsletter that they could promote the course in?
- Maybe a Joomla lover who has a Blog would like to Blog?
- Spread the word in a Joomla forum?
- Other?

**Would anyone who has some kind of history in the Joomla Community like to lend their weight?**

**Reply by sonvnn** [reply]
@paurray Thank you for your contribute. Please send me your email address. I will send you our Youtube Account, then you can upload your video to Astroid channel here https://www.youtube.com/channel/UCUHl1uU0Ofkyo-1ke-K4_xg

@Chacapamac I know you are busy right now. But if you have time, please help me complete this project with Paul. I don't have much experience with media and video projects. I have been spending all time and focus 100% to Astroid v3. You can check it here https://github.com/templaza/astroid-framework/tree/v3 . It is only 10% of the project, but Astroid will be a really new thing and extremely optimized in terms of performance.

Thank you all for cooperating! Have a great day.

Cheers,
Sonny

**Comment by paurray** [comment]
@Chacapamac @sonvnn 
This is how I would like to see it work:

**a)** **Astroid Channel Admin.** (Legal reasons*) Not me. **Inserts a Playlist on Astroid YT Channel**
(Videos are too large to send/would like to keep originals on my Chanel = My Copyright is clear.)

*Note 1
Yes this is paranoid. The reason being that I/we have no control over what other admins do with the Astroid YT Channel. It does not even have to be another admin doing something bad. It could be as banal as an admins access getting hacked. Something malicious happening and theoretically being as a fellow admin responsible/liable.

I guess the Channel Admin. Not me. Should be able to insert this*
https://youtube.com/playlist?list=PLOd7DxRu2yaJvVJKPEGZAOGoWpWOyNL1b

*Note 2
The playlist is as of now not published so we probably need to co-ordinate this!?!

My email is paul(at)finalbug.tv

Maybe try this first:

How to Create a YouTube Playlist on Your Channel
https://www.youtube.com/watch?v=mub4FbMSaPY

**b)** **The Text on the Astroid Playlist on the Astroid YT Channel should be as follows***

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Step by step instructions. How to build a website from scratch using Astroid and Joomla. Suitable for Novices.

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

**c)** **The Thumbnail for the Playlist should be the thumbnail from the 2nd video.**
As in rather than “Preamble way too Long” we see “Astroid Install First Baby Steps”
The playlist still starts with the First video with the Way to long warning and link to 2nd video.
**I have tried to take care of this at my end and it might not be an issue.**

https://www.youtube.com/watch?v=OVw5O8tZr9c
Youtube playlist thumbnail image -- how to change it (for Youtube playlists that you own)

**But please double check. That the playlist has the correct Thumbnail  before launch :-)**

**d)** **The links on individual Videos in the playlist need to ripple through on the Astroid YT Channel.**
01 Astroid : Preamble Way Too Long
https://youtu.be/JB8Fsn6nd-g
**I have tried to take care of this at my end and it might not be an issue.**

**But please double check. That they work in individual Videos in your playlist description before launch :-)**

Here is an eg for the first video. Most of the other videos have very few if any links:

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
01 Astroid : Preamble Way Too Long
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Warning! This video is kind of boring. I recommend looking at the chapters below! Or if you just want to jump straight in proceed to 2nd video in this Series HERE ➡️ https://youtu.be/dWu_j6xLO2Y

00:00 - Intro from the Edit Bay
00:20 - My long checkered Joomla past
02:06 - Figuring out that I was just over doing things ➡️  https://bit.ly/3QpETbh
02:45 - Starting my Joomla/Astroid research ➡️ https://bit.ly/47eCfuU

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Support the creator of these videos here:
https://www.buymeacoffee.com/paurray

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

03:24 - Check release Cycle before committing ➡️ https://bit.ly/3q6K70Z
03:54 - Setting a goal for my Astroid/Joomla project
04:19 - My early frustration with respect to training material
05:26 - Reaching out to the Astroid Community ➡️ https://bit.ly/3OCcttk
06:29 - Why these courses came to be
06:57 - What I would like to achieve
07:12 - Thank you & disclaimer ➡️ https://bit.ly/3Yjofff
07:51 - Astroid is Alive ➡️ https://bit.ly/3QsZSdr
08:12 - This is what we want to build
08:36 - A little bit more about me ➡️ https://finalbug.tv

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Get Astroid Help Here: ➡️ https://bit.ly/3OCcttk

- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

**e) I would like to agree a date for a launch eg next **full moon 29th of August.****

**f) Have things set up and ready to go **lets say by the middle of next week and start another thread** **where people can be asked in advance before the launch to spread the word. :-)****

**Edited**: I removed the Clickable playlist description thing. Do not think it made much sense!

**Comment by Chacapamac** [comment]
@paurray 
**Sorry I’m not to responsive these days, I’m busy. I will try to stay with you on that.**

1- Ok, if I understand well, you seem reluctant to put your videos in the **Astroid Youtube channel**. The idea here is to centralized the information on Astroid. You understand that, for Astroid, sending people all over the web will not look to professional and is somewhat confusing. 

• I think it is a way to embed your playlist (from your channel) in the Astroid Youtube Channel without Astroid having the ability to change your content and even, in a way, promote your channel and content.
**— I’m looking at the possibility and come back to you soon.**
If it’s no possibility or you are not Ok with that, Astroid can still have link to your content if needed.

I come back to You on that....

**Comment by paurray** [comment]
@Chacapamac 
Sorry for being so fussy but this is in every ones interests.
Eg I just bough the Astroid logo that I used on the Thumbnails.
Imagine if I had not.
Then uploaded the videos as an admin on the Astroid YT channel and did a disappearing act.
And then image if you will the company selling the logo found out and had an aggressive lawyer team…
Don’t get me wrong I love lawyer but when they are on my side giving me good advice.

I am pretty sure it is possible
How to Move a Playlist Between YouTube Channels
https://www.youtube.com/watch?v=z6WiPTuTg9E

Let's email.

If all else fails a link is also good.

**Reply by Chacapamac** [reply]
@paurray 
What you mean by _“I just bough the Astroid logo that I used on the Thumbnails.”_
You don’t need to buy it, I have all the logo right here, Completly free?

You said _“Then uploaded the videos as an admin on the Astroid YT channel and did a disappearing act.”_
You mean that you have access to the Astroid  Google account?, I don’t even have that?

The Astroid/name/image and code is all in the public domaine now, who sell you a Logo that don’t belong to them?

I Think a great way is to add your channel as a “**Featured Channel**” on the Dashboard Page of Astroid, Like that, for Astroid, we have more infos on Astroid and we advertise your Channel at the same time.

**Comment by paurray** [comment]
@Chacapamac 

I bought this for 10$
https://www.vecteezy.com/vector-art/8556563-vector-graphic-of-world-asteroid-day-good-for-world-asteroid-day-celebration-flat-design-flyer-design-flat-illustration
It is probably a rip off but paying the money for it gives me peace of mind.
I covered my rear end/acted in good will to the best of my abilities.

With respect to my copy right potential pit falls I was talking hypothetical.
I do not have access to the Astroid YT account and do not want it :-)

**Can you post a link to the “Featured page” you are talking about?**

PS” Logo” is also from my side technically the wrong word.
I am trying to create a colourful Thumbnail.
With an Astroid and planet = uncertainty.
And the question “A match made in heaven?”
Something that makes people curious....
Hope this now makes sense.

Edited to add PS

**Comment by Chacapamac** [comment]
**Featured Channels example:** https://www.youtube.com/channel/UCMKaNyOe0J9RlqbjcaiKr2A
![Screen Shot 2023-08-06 at 10 30 39 AM](https://github.com/templaza/astroid-framework/assets/1907276/0ae8dc65-43f4-43dd-b158-035b7ab7ab12)

**I will prepare a package with all logos and trademarks for collaborators**

**Comment by paurray** [comment]
I have done some more work on the “Astroid & Joomla Seamlessly” YT channel. And all videos are as of now published. I also added the official “Astroid” channel in the featured section. I hope this helps and please fell free to add me to, your YT channel if it works for you. Thank you @Chacapamac  and @sonvnn  for all the positive and constructive feedback.

https://www.youtube.com/channel/UCRZ-DzZsXU-CNYn0bNqGtBQ

I will probably keep modifying it but this is the current state....

<img width="960" alt="CurrentStateAandJ" src="https://github.com/templaza/astroid-framework/assets/9163424/752f000f-67bf-417d-af79-22297e3b24cc">
